### PR TITLE
Issue 8482: Add missing REQUIRED_IDENTIFIERS_MISSING message text

### LINKED ISSUE #8482: URBridgeEntity uses NLS message key, REQUIRED_IDENTIFIERS_MISSING, which is not defined
The URBridgeEntity class uses NLS message key, REQUIRED_IDENTIFIERS_MISSING, which is not defined. It results in an message such as the one below being produced, and is not helpful:

```
Caused by: com.ibm.wsspi.security.wim.exception.WIMApplicationException: REQUIRED_IDENTIFIERS_MISSING
	at com.ibm.ws.security.wim.adapter.urbridge.utils.URBridgeEntity.getUniqueId(URBridgeEntity.java:216)
	at com.ibm.ws.security.wim.adapter.urbridge.utils.URBridgeEntity.setIdentifierProperties(URBridgeEntity.java:190)
	... 136 more
```

The new message will look like the following:

```
Caused by: com.ibm.wsspi.security.wim.exception.WIMApplicationException: CWIML3001W: The required identifiers for the entity are missing.
	at com.ibm.ws.security.wim.adapter.urbridge.utils.URBridgeEntity.getUniqueId(URBridgeEntity.java:216)
	at com.ibm.ws.security.wim.adapter.urbridge.utils.URBridgeEntity.setIdentifierProperties(URBridgeEntity.java:190)
	... 136 more
```

### PR DESCRIPTION
**Pull request by kristip17**
Fixes #8482

Add the missing text for the REQUIRED_IDENTIFIERS_MISSING key. For some reason, this message was not moved over from WebSphere Application Server. Adding it in now.

### CODE REVIEW

**Comment by kristip17** [comment]
#build

File: dev/com.ibm.websphere.security.wim.base/resources/com/ibm/websphere/security/wim/util/resources/WebsphereWimUtilMessages.nlsprops [hunk 1]
```diff
@@ -26,6 +26,10 @@ DUPLICATE_PROPERTY_ENTITY=CWIML1997W: The extended property ''{0}'' is ignored b
 DUPLICATE_PROPERTY_ENTITY.explanation=Because the property is already defined in the entity's schema, it is ignored since it cannot be redefined as an extended property.
 DUPLICATE_PROPERTY_ENTITY.useraction=Check the entity's schema to determine if the property can be used. Either rename or delete the conflicting extended property from the server configuration.
 
+REQUIRED_IDENTIFIERS_MISSING=CWIML1998W: The required identifiers for the entity are missing.
```
[the remark this answers is not in the document]
Comment (jvanhill) [reply dev/com.ibm.websphere.security.wim.base/resources/com/ibm/websphere/security/wim/util/resources/WebsphereWimUtilMessages.nlsprops]: In tWAS we used CWWIML3001, so probably should use the same here. CWIML3000 to CWIML3499 are assigned to UserRegistry messages, which this falls in under. To make sure that the message is available, I ran my `findNextMessage` script to verify it was not used.

```
[jlvanhil@oc6220123152:dev (master $=)]$ findNextMessage.sh
Searching CWIML3000... AVAILABLE
Searching CWIML3001... AVAILABLE
Searching CWIML3002... AVAILABLE
Searching CWIML3003... AVAILABLE
Searching CWIML3004... AVAILABLE
Searching CWIML3005... AVAILABLE
Searching CWIML3006... 
Searching CWIML3007... AVAILABLE
Searching CWIML3008... AVAILABLE
Searching CWIML3009... AVAILABLE
Searching CWIML3010... 
Searching CWIML3011... 
Searching CWIML3012... 
Searching CWIML3013... AVAILABLE

Found the following free messages:  CWIML3000 CWIML3001 CWIML3002 CWIML3003 CWIML3004 CWIML3005 CWIML3007 CWIML3008 CWIML3009 CWIML3013
```

**Comment by andrea-pichler** [comment]
Hi @kristip17 
while doing the L2 message review I found that the REQUIRED_IDENTIFIERS_MISSING.explanation uses "uniqueName and uniqueId" while the REQUIRED_IDENTIFIERS_MISSING.useraction uses "unique name" and "unique Id". Please consider modifying these to either have a space or no space. Many thanks in advance!

**Comment by kristip17** [comment]
#build

**Pull request merged**